t/25-bugs.t

```diff
@@ -40,4 +42,9 @@ ok($bug->refreshed, 'bug refreshed');
 ok($bug->bugid,     'bugid matched');
 is($c->bugtitle_for('poo#200', $bug), "Bug referenced: poo#200\nfoo bar < \" & ß", 'bug title not already escaped');
 
+ok(scalar($bugs->all) > 0);
+my $job = Minion::Job->new(id => 42, minion => $t->app->minion, task => 'foo', app => $t->app);
+OpenQA::Task::Bug::Limit::_limit($job);
+ok(scalar($bugs->all) == 0, "Unreferenced bugs cleanup job works");
```

Comment:
Please use `is` instead of `ok` for the check. And it would be great to test the case when not every bug is deleted as well. E.g. just create a comment with a bugref before running `_imit` and check whether `$bugs->all` returns exactly the one bug which is still referenced.

Reply:
And it would be nice if you would make it a subtest. (Just grep for subtest in our testsuite for examples.)

Reply:
It's not hard to actually run the job properly.
```perl
my $minion = $t->app->minion;
my $id = $minion->enqueue('limit_bugs');
my $worker = $minion->worker->register;
my $job = $worker->dequeue(0, {id => $id});
$job->perform;
$worker->unregister;
```
Something like this should work.

Reply:
@kraih I think it is generally a good idea to keep it simple and therefore run unit tests with as less dependencies as possible.

The way you propose running the job doesn't make it much more complicated considering we use the database here anyways. However, that should be a generic test helper within the test utils and I wouldn't require it for this PR.

Reply:
too late - now I changed it.

Reply:
@Martchus The original test was rather unrealistic though. Using private methods to run the code in a completely different context than a production system would. And there is no guarantee whatsoever that a manually initialized `Minion::Job` object like that will keep working after the next Minion update (more meta data is added that job methods will depend on).

Reply:
That said, i wouldn't mind a test helper, we already have multiple implementations of this in the tests. Some pretty terrible actually, relying on the command system to trigger the one-shot feature for a random inactive job. 😄

Reply:
> Some pretty terrible actually, relying on the command system to trigger the one-shot feature for a random inactive job.

I know and therefore I didn't want @asdil12 to copy that. But let's keep it like this for now and extract it later.